### ISSUE TITLE
Name and ControllerHost printing is broken

### ISSUE THREAD

**Issue by Thermi** [coreos/kube-aws#341]
When running `./kube-aws up [...]"`:

```
Creating AWS resources. Please wait. It may take a few minutes.
Success! Your AWS resources have been created:
Name=, ControllerHost=
The containers that power your cluster are now being downloaded.

You should be able to access the Kubernetes API once the containers finish downloading.
```
Version v0.9.4-rc.3

**Comment by mumoshu** [comment]
Thanks @Thermi!
It seems like I've missed implementing it in #315

**Comment by mumoshu** [comment]
I'm closing this as fixed via #356 but please feel free to reopen if it still persists.
Thanks for reporting, @Thermi 👍
